plugins/bundle/plugin_test.go

```diff
@@ -408,8 +409,8 @@ func TestPluginListener(t *testing.T) {
 	go plugin.oneShot(ctx, bundleName, download.Update{})
 	s4 := <-ch
 
-	if !reflect.DeepEqual(s3, s4) {
-		t.Fatalf("Expected: %v but got: %v", s3, s4)
+	if s4.ActiveRevision != "fancybluederg" || s4.Code != "" || s4.Message != "" || len(s4.Errors) != 0 {
```

Comment:
I'm guessing the comparison is done this way to avoid failing on metric timers? Perhaps it would be clearer and more future proof to unset the metrics contained in the update?

Reply:
You are correct, and thats a good idea. This check is the same as what gets used above as those are the fields in particular that we are wanting to validate.. I'll go do the right thing and make a helper to validate them 😄